test/test-libsos.cc

```diff
@@ -0,0 +1,93 @@
+//
+//  main.cpp
+//  test-libsos
+//
+//  Created by Pavan Kumar Sunkara on 1/21/15.
+//  Copyright (c) 2015 Apiary Inc. All rights reserved.
+//
+
+#define CATCH_CONFIG_MAIN /// < Let catch generate the main()
+
+#include "catch.hpp"
+#include "sosJSON.h"
+#include "sosYAML.h"
+#include "test.h"
+
+TEST_CASE("Serailize JSON", "[sos][json]")
+{
+    std::stringstream output;
+    std::string expected = \
+    "{\n"\
```

Comment:
I think it would be better if these we're in separate files, it would make it easier to read and modify along with allowing them to easily be pulled out and linted with other tools.

So pulling them into files like the following:

- `test/fixtures/example.json`
- `test/fixtures/example.yaml`